### PR TITLE
Removing unneeded microservices

### PR DESCRIPTION
**Pull request by trevars**
Scaling down some Gen3s

https://ctds-planx.atlassian.net/browse/PXP-10664

### Environments
https://acct.bionimbus.org/
https://gen3.datacommons.io/
https://caninedc.org/
https://icgc.bionimbus.org/
https://genomel.bionimbus.org/

### Description of changes
Removing WTS, Hatchery, and access-backend from these environments. Reducing target CPU. Modifying FE to remove references to workspaces.

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io/manifest.json
## :warning: Services on branch
- frontend-framework

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io/manifest.json
## :warning: Services on branch
- frontend-framework

**Comment by PlanXCyborg** [comment]
# caninedc.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io/manifest.json
## :warning: Services on branch
- frontend-framework

**Comment by PlanXCyborg** [comment]
# caninedc.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io/manifest.json
## :warning: Services on branch
- frontend-framework

**Comment by PlanXCyborg** [comment]
# caninedc.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io/manifest.json
## :warning: Services on branch
- frontend-framework

**Comment by PlanXCyborg** [comment]
# caninedc.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# icgc.bionimbus.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io/manifest.json
## :warning: Services on branch
- frontend-framework

File: acct.bionimbus.org/manifest.json [hunk 1]
```diff
@@ -22,8 +22,6 @@
     "spark": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-spark:2023.01",
     "tube": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/tube:2023.01",
     "guppy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/guppy:2023.01",
-    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2023.01",
```
Comment (paulineribeyre) [inline acct.bionimbus.org/manifest.json:25]: we should probably remove the `manifests/hatchery/` directory for all of there too

File: gen3.datacommons.io/manifest.json [hunk 2]
```diff
@@ -359,7 +357,7 @@
       "strategy": "auto",
       "min": 1,
       "max": 2,
```
Comment (paulineribeyre) [inline gen3.datacommons.io/manifest.json:359]: should we also remove or update all the scaling sections to only deploy 1 pod for each service?

File: acct.bionimbus.org/manifest.json (hunk 1 above)
Comment (trevars) [reply acct.bionimbus.org/manifest.json:25]: Fair. ICGC & GenoMEL didn't have them, but added the other removals to this PR.

**Comment by PlanXCyborg** [comment]
# caninedc.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# icgc.bionimbus.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io/manifest.json
## :warning: Services on branch
- frontend-framework

**Comment by PlanXCyborg** [comment]
# caninedc.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# icgc.bionimbus.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io/manifest.json
## :warning: Services on branch
- frontend-framework

**Comment by PlanXCyborg** [comment]
# caninedc.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# icgc.bionimbus.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io/manifest.json
## :warning: Services on branch
- frontend-framework

**Comment by PlanXCyborg** [comment]
# caninedc.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# icgc.bionimbus.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [node_id] in explorerConfig.charts not found in etlMapping

**Comment by PlanXCyborg** [comment]
# gen3.datacommons.io/manifest.json
## :warning: Services on branch
- frontend-framework

File: gen3.datacommons.io/manifest.json (hunk 2 above)
Comment (trevars) [reply gen3.datacommons.io/manifest.json:359]: I think I'll scope that in for a future change to discuss with PE. Thanks for this, Pauline!

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**